clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/ProjectClustersService.java

```diff
@@ -383,7 +383,7 @@ public DeployedBuild(String host, String job, String buildNumber, Long deployed,
       this.job = job;
       this.buildNumber = buildNumber;
       this.deployed = deployed;
-      this.images = images;
+      this.images = (images == null) ? images : new ArrayList(images);
```

Comment:
If `images == null` would it make more sense to make it an empty array list instead of null?  (If it's null we'll probably get an NPE in the same place we were getting an exception modifying the immutable list.)

Reply:
Sounds good, let me make another commit with the change.